build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.0.6'
```

Comment:
is there any comptability issue for applications compiling with 1.0? I got a note in slack that it should be compiled with a flag (which I haven't found)